ui/ui-components/components/navigation/HighlightingNavLink.tsx

```diff
@@ -67,6 +81,28 @@ export default function HighlightingNavLink({
           </span>
         </a>
       </Link>
+      {active &&
+        navChildItems?.map((childNav) => (
+          <div key={childNav.href} className="ml-3">
+            <HighlightingNavLink
+              href={childNav.href}
+              mainPathSectionIdx={childNav.mainPathSectionIdx ?? 1}
+              text={
+                <>
+                  {childNav.title}
+                  {childNav.title === "Runs" ? (
+                    <>
+                      {" "}
+                      <RunningRunsBadge />
```

Comment:
Actually, this logic is still in the parent component: https://github.com/grouparoo/grouparoo/pull/2953/files#diff-f356a14f3edf616238baef014100f4a869eaa21d8bbf00df34305b2b420d3066L194

It got added here due to copy-paste, but after taking a second look it seems unnecessary to include it here. This code path handles the subNav items only, and since the "Runs" nav item is handled as a top level item behind the accordion, it doesn't really need to be here.